src/web/core/src/main/java/org/geoserver/web/services/BaseServiceAdminPage.java

```diff
@@ -383,4 +384,51 @@ public LocalWorkspacePanel(String id, T service) {
             add(new Label("workspace", new PropertyModel(service, "workspace.name")));
         }
     }
+
+    /**
+     * Override this method to return true if the implementation support international content
+     *
+     * @return true if support international content false otherwise.
+     */
+    protected boolean supportInternationalContent() {
+        return false;
+    }
+
+    public Fragment getTitleAndAbstractFragment(IModel<T> infoModel, String id) {
```

Comment:
Why public? Anyways, since this is a common combo used in other places as well, maybe it's best to create a larger component that does both in one shot?

Reply:
fixed the public. The component is common to all the service configuration page that have i18n support, while the component intended to be reusable by all is the InternationalStringPanel. The logic in this method is meant to handle the specific services case where not all the services have international support. Make another component handling the two scenario would make it still only usable by this class...